nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -26,6 +26,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - [fzf](https://github.com/junegunn/fzf), a command line fuzzyfinder. Available as [programs.fzf](#opt-programs.fzf.fuzzyCompletion).
 
+- [ydotool](https://github.com/ReimuNotMoe/ydotool), Generic command-line automation tool. Available as [programs.ydotool](#opt-programs.ydotool.enable).
```

Comment:
What does ``Generic command-line automation tool`` mean?

Reply:
I've taken the description from upstream. I think what they're trying to convey is that it works under wayland, xorg, fbdev, ..

I can of course change it if it's not adequate